swift_browser_ui_frontend/src/views/Containers.vue

```diff
@@ -512,10 +515,11 @@ export default {
           .modify({ [this.active.id]: { largeProjectNotification: true } });
       }
     },
-    toggleCreateFolderModal: function (folderName) {
-      this.$store.commit("toggleCreateFolderModal", true);
-      if (folderName) {
-        this.$store.commit("setFolderName", folderName);
+    viewFolder(event) {
+      // Prevent keyboard navigation from opening folder
+      // when accessing row menu
+      if (event.target.localName !== "c-menu") {
+        this.$router.push(this.getConAddr(this.selected["name"]));
```

Comment:
It looks like this function does not use anywhere else ?

Reply:
Good catch. This should be in use and is called now in where it's supposed to be.

Reply:
👍